client/main.js

```diff
@@ -73,11 +74,28 @@ $(document).ready(function () {
     // Makes an API request for a new song every TIME_INTERVAL milliseconds.
     let timeout = () => {
         activeInterval = true;
-        setTimeout(function () {
+        songTimeout = setTimeout(function () {
             loopPlayer(audioElement);
         }, TIME_INTERVAL);
+        startTimer();
```

Comment:
Please use `timer.startTimer()` (etc for all timer functions) so we can tell at a glance that the function is imported, rather than scrolling down looking for it.